features/raadpleeg-bewoning-met-periode/test/geboortedatum.feature

```diff
@@ -0,0 +1,41 @@
+            #language: nl
+
+            @api
+            Functionaliteit: raadpleeg bewoning in periode levert geboortedatum bewoner
```

Comment:
Ik denk dat het handig is om deze feature te verplaatsen naar de features map en dan een BewoningOpPeildatum scenario toevoegen.
Anders is er alleen gespecificeerd en gevalideerd dat geboortedatum wordt geleverd bij BewoningMetPeriode